Build/package.json

```
@@ -0,0 +1,103 @@
+{
+  "name": "kitodo-presentation",
+  "version": "5.0.4",
```

This will be part of v6.0.0.
```suggestion
  "version": "6.0.0",
```